source/Octopus.Cli/Commands/Package/PushMetadataCommand.cs

```diff
@@ -19,11 +19,11 @@ public PushMetadataCommand(IOctopusAsyncRepositoryFactory repositoryFactory, IOc
             : base(clientFactory, repositoryFactory, fileSystem, commandOutputProvider)
         {
             var options = Options.For("Package metadata pushing");
-            options.Add("package-id=", "The ID of the package, e.g., 'MyCompany.MyApp'.", v => PackageId = v);
-            options.Add("version=", "The version of the package; defaults to a timestamp-based version", v => Version = v);
-            options.Add("metadata-file=", "Octopus Package metadata Json file.", file => MetadataFile = file);
-            options.Add("overwrite-mode=", "If the package metadata already exists in the repository, the default behavior is to reject the new package metadata being pushed (FailIfExists). You can use the overwrite mode to OverwriteExisting or IgnoreIfExists.", mode => OverwriteMode = (OverwriteMode)Enum.Parse(typeof(OverwriteMode), mode, true));
-            options.Add("replace-existing", "If the package metadata already exists in the repository, the default behavior is to reject the new package metadata being pushed. You can pass this flag to overwrite the existing package metadata. This flag may be deprecated in a future version; passing it is the same as using the OverwriteExisting overwrite-mode.", replace => OverwriteMode = OverwriteMode.OverwriteExisting);
+            options.Add<string>("package-id=", "The ID of the package, e.g., 'MyCompany.MyApp'.", v => PackageId = v);
+            options.Add<string>("version=", "The version of the package; defaults to a timestamp-based version", v => Version = v);
+            options.Add<string>("metadata-file=", "Octopus Package metadata Json file.", file => MetadataFile = file);
+            options.Add<OverwriteMode>("overwrite-mode=", "If the package metadata already exists in the repository, the default behavior is to reject the new package metadata being pushed (FailIfExists). You can use the overwrite mode to OverwriteExisting or IgnoreIfExists.", mode => OverwriteMode = mode);
```

Comment:
Should the list of valid values be generated from the enum now? E.g. https://github.com/OctopusDeploy/OctopusCLI/blob/7cbb503ac8720481735bf62b5b004719a5722178/source/Octopus.Cli/Commands/InstallAutoCompleteCommand.cs#L25